pkg/tangle/solidifier.go

```diff
@@ -283,8 +294,16 @@ func (t *Tangle) solidifyMilestone(newMilestoneIndex milestone.Index, force bool
 
 	var timeStartConfirmation, timeSetConfirmedMilestoneIndex, timeUpdateConeRootIndexes, timeConfirmedMilestoneChanged, timeConfirmedMilestoneIndexChanged, timeMilestoneConfirmedSyncEvent, timeMilestoneConfirmed time.Time
 
+	parentsTraverser := dag.NewParentsTraverser(memcachedTraverserStorage)
+
 	timeStart := time.Now()
-	confirmedMilestoneStats, confirmationMetrics, err := whiteflag.ConfirmMilestone(t.storage, t.serverMetrics, messagesMemcache, metadataMemcache, cachedMsToSolidify.Milestone().MessageID,
+	confirmedMilestoneStats, confirmationMetrics, err := whiteflag.ConfirmMilestone(
+		t.storage.UTXOManager(),
+		parentsTraverser,
+		metadataMemcache.CachedMessageMetadata,
```

Comment:
we could pass a traverser storage instead of the traverser and the 2 funcs.

Reply:
Good idea, will change that!